docs/reference/plugins.md

```diff
@@ -1 +1,16 @@
 # Plugins
+
+Shift supports the integration of custom Plugins. Shift Plugins are Qt based UIs which can interact with Shift to display additional information, perform actions or modify the behaviour of the application.
+
+## Shift Native Plugins
+
+Shift ships with two example plugins which can be found in the *Tools* section in the top menu bar. The desired plugin can be opened by clicking on the corresponding entry.
+Once a Plugin is opened, the Shift UI can be customized with the best suited layout.

```

Comment:
Shouldn't we mention this in the description of plugins, instead of the Shift Native Plugins section?
Basically, as a general rule for all, Plugins are added as dockable widgers in the ShiftUI, meaning they can be resized and placed as desired.

Reply:
moved this part above as you suggested